inference-engine/ie_bridges/c/src/ie_c_api.cpp

```diff
@@ -335,6 +336,64 @@ IEStatusCode ie_core_read_network_from_memory(ie_core_t *core, const uint8_t *xm
     return status;
 }
 
+IEStatusCode ie_core_import_network(ie_core_t *core, const char *file_name, const char *device_name, \
```

Comment:
Why do you need `\` symbol at the end? This is not a comment/string continuation.

Reply:
copy/pasted issue) fixed